### PR TITLE
Bug fix system specs refs #436

### PR DESCRIPTION
**Pull request by miosidler**

### CODE REVIEW

File: spec/system/ajax_hook_system_spec.rb [hunk 1]
```diff
@@ -10,13 +10,14 @@
 describe 'AjaxHook', type: :system, js: true do
   include SystemHelpers
 
-  it 'show user has role user message' do
+  it 'show user has role user' do
```
Comment (luca-bruegger) [inline spec/system/ajax_hook_system_spec.rb:13]: 'shows user has default user role'

File: spec/system/teammember_system_spec.rb [hunk 2]
```diff
@@ -59,7 +59,6 @@
     # click_link 'Api Users'
     #
     # within('#api-users') do
-    #   require 'pry'; binding.pry;
```
Comment (luca-bruegger) [inline spec/system/teammember_system_spec.rb:62]: Are those commented lines needed? Maybe delete them? 😉

File: spec/system/teams_page_system_spec.rb [hunk 3]
```diff
@@ -14,15 +14,13 @@
   include SystemHelpers
 
   it 'clicks through teams page' do
+
```
Comment (luca-bruegger) [inline spec/system/teams_page_system_spec.rb:17]: Blank line

**Review by luca-bruegger — CHANGES_REQUESTED** [review]
Some small things to fix - nice that those tests are up & running again 🚀

**Review by luca-bruegger — APPROVED** [review]
Looks fine for me now! 🚀

**Review by TheWalkingLeek — APPROVED** [review]
Looking good! 👍🏼

**Review by TheWalkingLeek — CHANGES_REQUESTED** [review]
Yikes, seems to have broken again after rebasing :/

**Pull request merged**